Lib/test/test_sysconfig.py

```diff
@@ -622,7 +623,7 @@ def test_abiflags(self):
         if os.name == 'nt':
             self.assertEqual(abiflags, '')
             # Example values: '', 't', 't_d', '_d'
-            self.assertLessEqual(ABIFLAGS.count('_'), 1)
+            self.assertTrue(ABIFLAGS.count('_d') == 1 or '_' not in ABIFLAGS, ABIFLAGS)
```

Comment:
Honestly, you may as well just check that it's one of the four values. The logic in the assertion is more complicated than the test is worth.